### PR TITLE
Added new plushie toy!

### PR DESCRIPTION
**Pull request by Stanislav4ix**
## About the PR
Added new plushie toy just for fun. You can scale it and do something fun :)

## Why / Balance
Because we need more toys!!

## Technical details
Nothing special.

## Media

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl: Stanislav4ix
- add: Added new toy! Meet "Plushie bubble"!

https://github.com/space-wizards/space-station-14/assets/124695665/3b296aeb-7a53-47ff-9380-fb0d6c119307

### CODE REVIEW

**Comment by deltanedas** [comment]
minifauna plushie when

**Comment by Stanislav4ix** [comment]
> minifauna plushie when

But I didn't even think about that, thanks for the idea bro :)

**Comment by Errant-4** [comment]
Either add some changelog text or remove the blank entry

**Comment by Stanislav4ix** [comment]
> Either add some changelog text or remove the blank entry

Changelog isn't blank

**Comment by deltanedas** [comment]
videos arent put in changelog
your cl is `add:`

**Comment by Stanislav4ix** [comment]
> 
> 
> 
> ## About the PR
> Added new plushie toy just for fun. You can scale it and do something fun :)
> 
> ## Why / Balance
> Because we need more toys!!
> 
> ## Technical details
> Nothing special.
> 
> ## Media
> 
> 
> - [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame
> https://github.com/space-wizards/space-station-14/assets/124695665/3b296aeb-7a53-47ff-9380-fb0d6c119307
> 
> **Changelog**
> :cl: Stanislav4ix
> - add:  Added new toy! Meet "Plushie bubble"!
> 
> 
>

**Comment by Stanislav4ix** [comment]
> 
> 
> 
> ## About the PR
> Added new plushie toy just for fun. You can scale it and do something fun :)
> 
> ## Why / Balance
> Because we need more toys!!
> 
> ## Technical details
> Nothing special.
> 
> ## Media
> 
> 
> - [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
> 
> **Changelog**
> :cl: Stanislav4ix
> - add: 
> 
> https://github.com/space-wizards/space-station-14/assets/124695665/3b296aeb-7a53-47ff-9380-fb0d6c119307
> 
> **Changelog**
> :cl: Stanislav4ix
> - add: Added new toy! Meet "Plushie bubble"!
> 
> 
>

**Comment by Stanislav4ix** [comment]
> 
> 
> 
> ## About the PR
> Added new plushie toy just for fun. You can scale it and do something fun :)
> 
> ## Why / Balance
> Because we need more toys!!
> 
> ## Technical details
> Nothing special.
> 
> ## Media
> 
> 
> - [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
> 
> **Changelog**
> :cl: Stanislav4ix
> - add: 
> 
> https://github.com/space-wizards/space-station-14/assets/124695665/3b296aeb-7a53-47ff-9380-fb0d6c119307
> 
> **Changelog**
> :cl: Stanislav4ix
> - add: Added new toy! Meet "Plushie bubble"!
> 
> 
>

**Comment by Stanislav4ix** [comment]
> videos arent put in changelog
> your cl is `add:`

Thanks, I fixed it

**Comment by Errant-4** [comment]
Dragon Roar might be a risky sound effect to make this spammable
I don't know about sprite copyright but should that part not be updated in meta.json ?

**Comment by deltanedas** [comment]
give it long cooldown like nuke toy

**Comment by Stanislav4ix** [comment]
> give it long cooldown like nuke toy

I did it, now cooldown is 30 seconds

**Comment by Errant-4** [comment]
Only the in-hand use is restricted, it can be thrown for continuous roaring. Also, when it hits a wall it plays both a squeak and a roar

File: Resources/Textures/Objects/Fun/toys.rsi/meta.json
Comment (ficcialfaint) [inline Resources/Textures/Objects/Fun/toys.rsi/meta.json:1]: Add info in `copyright` field about where you took the sprite form

**Review by ficcialfaint — CHANGES_REQUESTED** [review]

**Comment by LankLTE** [comment]
@Stanislav4ix Are you coming back to this?

**Comment by LankLTE** [comment]
This could probably serve to be made obtainable somewhere- the arcade cabinet rewards or the plushie crate are usual spots. That said I don't believe it's necessary.

**Comment by TheShuEd** [comment]
looks like you can't get that toy anywhere. Is this planned, or did you forget to add her ways to get it?

**Comment by metalgearsloth** [comment]
Re-open if you come back to it.

**Pull request closed without merging**